include/inexor/vulkan-renderer/wrapper/device.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "inexor/vulkan-renderer/exception.hpp"
```

Comment:
Oh. This should be in the .cpp file.

Reply:
Ok so it's already in the .cpp, so since it's not used in the header I will remove it.

Reply:
done.